src/update_lmod_system_cache_files.in

```diff
@@ -334,7 +334,7 @@ set_defaults
 check_parameters
 
 if [ -z "$LMOD_REVERSEMAPT_DIR" ]; then
-  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/../reverseMapD"
+  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/reverseMapD"
```

Comment:
OK, we've disable the reversedb cache for now. My main grip with this is that if you want some different from the default, you again need to know the location of the Lmod cache. Maybe we can let you specific the reversedb cache location relative to the Lmod cache?